### ISSUE TITLE
[ARCHETYPE-681] [REGRESSION] Output from archetype verification is no longer logged

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-archetype#866]
**[Wolfgang Knauf](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=wknauf)** opened **[ARCHETYPE-681](https://issues.apache.org/jira/browse/ARCHETYPE-681?redirect=false)** and commented

When build an archetype project, there is a step the generates a sample project from the archetype.

 

With 3.1.2, the maven output to create and build a sample project from the archetype was logged (see duplicated "[INFO] [INFO]" lines):

 

`[INFO] Project created from Archetype in dir: C:\Temp\github\wildfly-archetypes\wildfly-jakartaee-webapp-archetype\target\test-classes\projects\multi\project\multi`
`[INFO] Invoking post-archetype-generation goals: verify`
`[INFO] [INFO] Error stacktraces are turned on.`
`[INFO] [INFO] Scanning for projects...`
`[INFO] [INFO]`
`[INFO] [INFO] -------------------------{-}{{-`< foo.bar:multi >`-`{-}--------------------------}}
`[INFO] [INFO] Building multi 0.0.1-SNAPSHOT`
`[INFO] [INFO]   from pom.xml`
`...`

 

With 3.3.0, there is just this output:

`[INFO] Project created from Archetype in dir: C:\Temp\github\wildfly-archetypes\wildfly-jakartaee-webapp-archetype\target\test-classes\projects\multi\project\multi`
`[INFO] Invoking post-archetype-generation goals: verify`
`[INFO] Post-archetype-generation invoker exit code: 0`
`...`

 

This is not good, as building the sample project might take a long time, or might even fail, and you don't see a reason for the long delay after the line "Invoking post-archetype-generation goals: verify" is printed. I had also a situation where creating and building the sample project failed, and here not details about the error were logged.

Could you bring back the old behavior to log the INFO output?

 

To reproduce: you could pull the wildfly archetypes: https://github.com/wildfly/wildfly-archetypes

At the end of the root pom.xml, change the archetype version to "3.3.0" - we are currently stuck with 3.1.2 due to another issue.

Then build one of the archetype modules (invoke it in the root directory - building all archetypes will not work, as one has additional requirements):

`mvn -pl wildfly-jakartaee-webapp-archetype clean install`

 

Or is there a configuration  option to enable logging again?

---

**Affects:** 3.3.0

**Remote Links:**
- [PR #227
](https://github.com/apache/maven-archetype/pull/227)

**Comment by jira-importer** [comment]
**[Slawomir Jaranowski](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=slawekjaranowski)** commented

There is an option `streamLogs` but logs are not forwarded to Maven build log so it is not working ... I will fix it.

**Comment by jira-importer** [comment]
**[Slawomir Jaranowski](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=slawekjaranowski)** commented

[Wolfgang Knauf](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=wknauf)

> At the end of the root pom.xml, change the archetype version to "3.3.0" - we are currently stuck with 3.1.2 due to another issue.

What another issue blocking you for upgrade?

**Comment by jira-importer** [comment]
**[Wolfgang Knauf](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=wknauf)** commented

Thanks for fixing this issue!

[sjaranowski](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=sjaranowski) I am blocked by https://issues.apache.org/jira/browse/ARCHETYPE-649 (seems to be a false warning)

**Comment by jira-importer** [comment]
**[Wolfgang Knauf](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=wknauf)** commented

I can confirm that it works again. Sorry for the verification delay ;)
